source/antwerp/remote_visualization_uantwerp.rst

```diff
@@ -142,243 +26,240 @@ section for information about ssh clients for various client operating systems.
 
-*Note that the first time that you start a Xfce session with TurboVNC,
-you'll see a panel "Welcome to the first start of the panel". Please
-select "Use default config" as otherwise you get a very empty
-desktop.*
+   .. note::
+      The first time that you start a Xfce session with TurboVNC,
+      you'll see a panel "Welcome to the first start of the panel". 
+      You may want to select "Use default config", as otherwise you 
+      get a very empty desktop.
 
 Step 3: Starting an application
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-#. Open a terminal window (if one was not already created when you
-   started your session).
-   In the default Xfce-environment, you can open a terminal by selecting
+1. Open a terminal window in the VNC session.
+   In the default Xfce-environment, you can open a terminal window by selecting
    \"Terminal Emulator\" in the \"Applications\" menu in the top left.
    The first time it will let you choose between selected terminal
    applications.
-#. Load the modules that are required to start your application of
+2. Load the modules that are required to start your application of
    choice.
-#. 2D applications or applications that use a software renderer for 3D
+3. 2D applications or applications that use a software renderer for 3D
    start as usual. However, to start an application using the
    hardware-accelerated OpenGL, you'll need to start it through
-   ``vglrun``. Usually adding ``vglrun`` at the start of the command
-   line is sufficient.
-   This however doesn't work with all applications. Some applications
-   require a special setup.
+   ``vglrun``. This will use `VirtualGL`_ to redirect the OpenGL calls
+   to your VNC server's display. 
 
-   #. MATLAB: start MATLAB with the ``-nosoftwareopengl`` option to
-      enable accelerated OpenGL:
+   Adding ``vglrun`` at the start of the command line is usually sufficient.
+
+   .. note:: For a quick test of your setup, enter
 
       .. code:: bash
 
-         vglrun matlab -nosoftwareopengl
-      
-      The MATLAB command ``opengl info`` will then show that you are
-      indeed using the GPU.
+         vglrun glxinfo
+         vglrun glxgears
+
+      The first command will print some information about the OpenGL
+      functionality that is supported. The second command will display a set
+      of rotating gears. Don't be fooled if they appear to stand still but
+      look at the \"frames per second\" printed in the terminal window.
+
+   However, this doesn't work with all applications. Some applications, like MATLAB,
+   might require a special setup.
 
-#. When you've finished, don't forget to log out in the Xfce desktop 
+   |Example| Start MATLAB with hardware-accelerated OpenGL with ``-nosoftwareopengl`` as follows:
+
+    .. code:: bash
+ 
+       vglrun matlab -nosoftwareopengl
+    
+    The MATLAB command ``opengl info`` will then show that you are
+    indeed using the GPU.
```

Comment:
There is a formatting issue here with the indentation:
```suggestion
   |Example| Start MATLAB with hardware-accelerated OpenGL with ``-nosoftwareopengl`` as follows:

   .. code:: bash
 
      vglrun matlab -nosoftwareopengl
    
   The MATLAB command ``opengl info`` will then show that you are
   indeed using the GPU.
```

Reply:
This was also indented extra for the bar to the left, keeping the example together.

Is there another way to separate this example cleanly from the rest of the instructions, or would you say this is OK (as it is quite a small example indeed)?